### PR TITLE
Options page redesign

### LINKED ISSUE #93: Update the design of Options page
The current design of Options page uses 3D styles for inputs, which are hard to style on Firefox and Edge browsers.

Also, the main panel for navigation takes a lot of space and could be further improved.

### PR DESCRIPTION
**Pull request by tomasko126**
Fixes #93 

Options page has been slightly redesigned.

### CODE REVIEW

File: options/index.html [hunk 1]
```diff
@@ -5,55 +5,44 @@
...
-        <script src="/lib/punycode.min.js"></script>
-        <script src="/lib/jquery.min.js"></script>
-        <script src="/js/port.js"></script>
-        <script src="/js/functions.js"></script>
-        <script src="/js/checkupdates.js"></script>
+        <script src="../lib/punycode.min.js"></script>
+        <script src="../lib/jquery.min.js"></script>
+        <script src="../js/port.js"></script>
+        <script src="../js/functions.js"></script>
+        <script src="../js/checkupdates.js"></script>
```
Comment (itskdog) [inline options/index.html:12]: How does this change things? Does it cause problems in some browsers?

File: options/index.html [hunk 2]
```diff
@@ -5,55 +5,44 @@
...
-                    <a data-scripts="/options/filters.js" data-page="filters" i18n="tabfilterlists"></a>
-                    <a data-scripts="/options/catblock.js" data-page="catblock">CatBlock</a>
-                    <a data-scripts="/filtering/domainset.js /filtering/filternormalizer.js /filtering/filtertypes.js /filtering/filteroptions.js /options/customize.js" data-page="customize" i18n="tabcustomize"></a>
-                    <a data-scripts="/options/support.js" data-page="support" i18n="tabsupport"></a>
+                    <a data-scripts="options/general.js" data-page="general" i18n="tabgeneral"></a>
+                    <a data-scripts="options/filters.js" data-page="filters" i18n="tabfilterlists"></a>
+                    <a data-scripts="options/catblock.js" data-page="catblock">CatBlock</a>
+                    <a data-scripts="filtering/domainset.js filtering/filternormalizer.js filtering/filtertypes.js filtering/filteroptions.js options/customize.js" data-page="customize" i18n="tabcustomize"></a>
+                    <a data-scripts="options/support.js" data-page="support" i18n="tabsupport"></a>
```
Comment (itskdog) [inline options/index.html:36]: Why was the forward slash removed? Doesn't that ensure it goes to the root of the extension's file tree?

**Comment by itskdog** [comment]
Just found a bug:
* Adding a new flickr photoset or search doesn't do anything. This is a regression as it works in the production version

File: options/index.html (hunk 2 above)
Comment (tomasko126) [reply options/index.html:36]: The forward slash was removed, when I have created a better algorithm for loading scripts.
The loading of those scripts is handled in the ```index.js``` file.

File: options/index.html (hunk 1 above)
Comment (tomasko126) [reply options/index.html:12]: There was a problem loading those scripts under Safari when using only slash at the beginning. This fixes it.

**Comment by tomasko126** [comment]
@kpeckett I am not able to confirm that bug on Chrome Canary on Mac, unfortunately. Can you still confirm this behaviour?

**Review by itskdog — APPROVED** [review]

**Pull request merged**